presets/dracula.kthm

```diff
@@ -0,0 +1,21 @@
+[General]
+background_color=#282a36
+blockquote_color=#6272a4
+cross_scheme_link_color=#ff5555
+external_link_color=#ff79c6
+external_link_prefix="\x21d2 "
+h1_color=#50fa7b
+h1_font="Sans Serif,20,-1,5,50,0,0,0,0,0"
+h2_color=#8be9fd
+h2_font="Sans Serif,15,-1,5,50,0,0,0,0,0"
+h3_color=#bd93f9
+h3_font="Sans Serif,12,-1,5,50,0,0,0,0,0"
+internal_link_color=#ff79c6
+internal_link_prefix="\x2192 "
+margins=55
+name=Default
```

Comment:
You should probably change this name to 'Dracula' so it doesn't override the Default theme ;)